### DISCUSSION TITLE
Convert from global to local coordinates.

### DISCUSSION THREAD

**Discussion by farazkhalidsiddiqui** [IfcOpenShell/IfcOpenShell#4024] [Q&A]
We can convert .ifc to .glb with “-building-local-placement” command to convert global coordinates to local coordinates. Is there a way to do the same conversion from .ifc to .ifc?
So the .ifc file replaces its global coordinates to local coordinates? 

Can the coordinates be converted from global to local for a particular type and its related objects from the IFC. For instance converting the coordinates for "ifcElementAssembly" mentioned in the below example.

Example:
	ifcElementAssembly = ifcFile.by_type("IfcElementAssembly")

**Comment by aothms** [comment]
No, the conversion from IFC to IFC is actually unaffected by any setting. We should probably try and make this more clear, but this a general issue that the applicability of command line flag to serializer output is not always clear. In v0.8 we'll try to make the definition of the settings more local to the respective serializers so that the code path of parsing the settings is actually also influenced by the selected output.

For now it'd advise to look at some of the ifcpatch recipes such as https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/ifcpatch/ifcpatch/recipes/ResetAbsoluteCoordinates.py to see if there is something covering your needs. I'm not fully familar with the set of recipes.
